Coordinator lookup join docs (#154524)
elastic/elasticsearch: docs/reference/query-languages/esql/esql-cross-serverless-projects.md

### LOOKUP JOIN across projects

{{esql}} `LOOKUP JOIN` follows the same constraints as [{{esql}} cross-cluster `LOOKUP JOIN`](/reference/query-languages/esql/esql-lookup-join.md#cross-cluster-support). By default, {{esql}} resolves the lookup index on every project in the query and each project joins against its own local index with that name. In this case, the lookup index must exist on every project being queried.

> If the lookup index is missing from one or more linked projects, use [coordinator mode](/reference/query-languages/esql/esql-lookup-join.md#coordinator-mode) in order to join against origin project lookup index copy.

Reviewer: are we missing an article here?

```suggestion
If the lookup index is missing from one or more linked projects, use [coordinator mode](/reference/query-languages/esql/esql-lookup-join.md#coordinator-mode) to join against an origin project lookup index copy.
```
